packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -826,11 +826,13 @@ export class ApiGateway {
 
   protected readiness: RequestHandler = async (req, res) => {
     let health: 'HEALTH' | 'DOWN' = 'HEALTH';
-
+    
     if (this.standalone) {
       const orchestratorApi = await this.adapterApi({});
 
       try {
+        // todo: test other data sources
+        orchestratorApi.seenDatasources = { default: true };
```

Comment:
It's better to use `orchestratorApi .addDataSeenSource('default');`

Weird about the moment when user is using multiple datasources without default.